Felienne/hedy
# [UI FIX] Fixes some minor UI bugs on for-teachers page

**Description**
In this PR we fix some minor UI bugs on the for-teachers page. We add the `cursor-pointer` class to the `a` elements without a `href` attribute to make sure we give the user feedback that the attribute is clickable. We also remove the underline on the "Remove" icon as well as the "View" string.


**Fixes**
This PR fixes #3097.

**How to test**
Verify that the UI elements are all changed according to the above stated changes. Also verify that the functionality is still as expected.
